app/email.go

```diff
@@ -276,6 +276,7 @@ func (a *App) SendInviteEmails(team *model.Team, senderName string, invites []st
 			props["display_name"] = team.DisplayName
 			props["name"] = team.Name
 			props["time"] = fmt.Sprintf("%v", model.GetMillis())
+			props["invite_id"] = team.InviteId
```

Comment:
I'm working on a unit test to ensure that `SendInviteEmails(...)` always includes the invite id in the URL that it generates, but it's a difficult bit of code to test because it actually sends emails. For the time being, manual testing will have to suffice.